_includes/accessibility.html

```diff
@@ -1,18 +1,28 @@
     <li><strong>Don’t control element order with CSS.</strong> Display form controls in the same order in the HTML as they appear on screen. Don’t use CSS to rearrange the form controls. Screen readers narrate form elements in the order they appear in the HTML.</li>
     <li><strong>Align validation with inputs.</strong> Visually align validation messages with the input fields so people using screen magnifiers can read them quickly.</li>
     <li><strong>Use proper markup.</strong> Group each set of thematically related controls in a fieldset element. Use the legend element to offer a label within each one. The fieldset and legend elements make it easier for those who use screen readers to navigate the form.</li>
     <li><strong>Use legends.</strong> Use a single legend for fieldset (this is required). One common use of the fieldset and legend elements is a question with radio-button options for answers. The question text and radio buttons are wrapped in a fieldset, with the question itself being inside the legend tag.</li>
     <li><strong>Embed multiple fieldsets and legends for more complex forms.</strong></li>
     <li><strong>Use simple vertical layouts.</strong> Keep your form blocks in a vertical pattern. This approach is ideal, from an accessibility standpoint, because of limited vision that makes it hard to scan from right to left.</li>
   </ul>
-
-  <div class="site-note"><strong>Note:</strong> These components have been designed to support a wide range of screen readers, but they may not work with all versions.</div>
+  <h2 id="general-accessibility-guidance-for-forms" class="usa-heading heading-margin-alt">Usability guidance</h2>
+  <ul class="usa-content-list">
+    <li><strong>Show whether a field is required or optional.</strong> Mark required fields as required by using a red asterisk  <font color="red"> *</font></li>
+    <li><strong>Use text descriptions.</strong> Combine red asterisks <font color="red"> * </font> with a text description at the top of the form instructing the user of its meaning. *Example: “A red asterisk <font color="red"> * </font> indicates a required field.”*
+    </li>
+    <li><strong>Label required fields as 'required'. </strong> Add the `required` attribute to any specific form elements that are required to let users of assistive technologies know this is a required field.</li>
+    <li><strong>Mark optional fields as "optional."</strong> Label optional fields with the word “optional” placed in parentheses.
+      *Example: (optional)*
+      </li>
+      <li><strong>Let users know if there is an error.</strong>Error handling: Use a message or alert to notify the user if a field that is required was not completed, and indicate which field was not completed. Additionally, include an error notification in the page title (`title` element) for screen readers.</li>
```

Comment:
Added space before `Error handling:` and wrapped `title` in code tag. 

![image](https://user-images.githubusercontent.com/3385219/196257606-81b26ac2-a40f-4c78-9de9-f6666f51da81.png)


```suggestion
      <li><strong>Let users know if there is an error.</strong> Error handling: Use a message or alert to notify the user if a field that is required was not completed, and indicate which field was not completed. Additionally, include an error notification in the page title (<code>title</code> element) for screen readers.</li>
```

Reply:
@bonnieAcameron Can we remove `Error handling:` from this copy? Feels like a different format than we use in the other copy.

Reply:
@mejiaj @bonnieAcameron I also wonder if it would be good to link to the the alert component here (assuming that using that component is our recommended practice - it might not be!)

Reply:
What do you think about Amy's suggestion, @jaclinec ?